frontend/src/mixins/UsersByRole.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Vue } from "vue-property-decorator";
+import fb from "@/firebaseApp";
+import { filteredStatus } from "@/utility";
+
+@Component
+export default class UserByRole extends Vue {
+  users: any[] = [];
+  usersBindName: string = "users";
+  usersRole: string = "";
```

Comment:
Maybe better to set it to `null` as a default value?

Reply:
It seems much easier to deal with the falsy value of the type rather than null. For example with the empty array this line `v-if="tasks && tasks.length"` becomes `v-if="tasks.length"`